Scripts/Linux/DownloadGVFSGit.sh

```diff
@@ -0,0 +1,7 @@
+. "$(dirname ${BASH_SOURCE[0]})/InitializeEnvironment.sh"
+
+BUILDDIR=$VFS_OUTPUTDIR/GVFS.Build
+GITVERSION="$($VFS_SCRIPTDIR/GetGitVersionNumber.sh)"
+cp $VFS_SRCDIR/nuget.config $BUILDDIR
+dotnet new classlib -n Restore.GitInstaller -o $BUILDDIR --force
+dotnet add $BUILDDIR/Restore.GitInstaller.csproj package --package-directory $VFS_PACKAGESDIR GitForMac.GVFS.Installer --version $GITVERSION
```

Comment:
This will download the nupkg containing a dmg; we'll need to adjust this and/or make our own GitForLinux installer with the gvfs patches?

Reply:
Yes ... the required effort here (for Linux) remains murky to me.  The existing CI jobs already build a Linux version of the patched GitForWindows (from microsoft/git), which is nice.  But we'll need to review how we want to package, both for CI and for customers, VFSForGit and the patched GitForWindows.